src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -202,6 +202,31 @@ private void initDragAndDrop() {
                     .filter(node -> node.getStyleClass().contains("tab-header-area"))
                     .findFirst()
                     .orElseThrow();
+            // Add drag and drop listeners to JabRefFrame
+            this.getScene().setOnDragOver(event -> {
```

Comment:
I think the similar code for the `tabHeaderArea` below can now be removed, since this is handled on the scene level. Right?

Reply:
Yes you are right, I have deleted the unnecessary code.